packs/data/equipment-effects.db/effect-choker-arm-mutagen-greater.json

```diff
@@ -0,0 +1,88 @@
+{
+    "_id": "HlUL7FmGLyS35Jpp",
+    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "name": "Effect: Choker-Arm Mutagen (Greater)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Choker-Arm Mutagen (Greater)]{Choker-Arm Mutagen (Greater)}</p>\n<p><strong>Implemented Effects</strong></p>\n<ul>\n<li>+10 foot reach</li>\n<li>+3 item bonus to Acrobatics checks to @UUID[Compendium.pf2e.actionspf2e.Escape]{Escape}, @UUID[Compendium.pf2e.actionspf2e.Squeeze]{Squeeze}, and @UUID[Compendium.pf2e.actionspf2e.Tumble Through]{Tumble Through}</li>\n<li>-1 penalty to Athletics checks, Stealth checks, Thievery checks, and attack rolls</li>\n<li>-1 penalty per damage die to all weapon and unarmed attack damage.</li>\n</ul>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "hours",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "system.attributes.reach.base",
+                "value": 10
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "system.attributes.reach.manipulate",
```

Comment:
I don't believe this one is needed, as it's implicit with the .base above.

Reply:
When I left it at just increasing the .base reach, I didn't seen an automatic increase in the manipulate reach.
Though tbh I don't know where manipulate reach is even used in this system.

Reply:
This is what the fields look like with the current "Greater" effect on:
![image](https://github.com/foundryvtt/pf2e/assets/69116996/efb55dfa-b522-4fef-93ea-425e9f9c1617)
So it doesn't look like .manipulate is being automatically increased with .base.

Reply:
When I try it, it looks like it's being automatically increased?

Reply:
...I stand corrected. It just so happens that increasing .manipulate at the same time didn't double-increase it. I'll fix that tomorrow then, sorry and thank you.

Reply:
Fixed with https://github.com/foundryvtt/pf2e/pull/7910/commits/5c181374c41214184b5ff6ac4d631db62a9abf05